conan/api/subapi/new.py

```diff
@@ -84,12 +84,20 @@ def _read_files(self, template_folder):
 
         return template_files, non_template_files
 
+    @staticmethod
+    def render_requires(requires):
```

Comment:
Why not using a ``for`` loop in the template directly? This seems a bit hybrid, doing some rendering here and some in the template?

Reply:
You're right. I've moved the rendering to jinja and kept only the logic here, good suggestion!